cli/src/main/java/org/jboss/pnc/projectmanipulator/cli/Cli.java

```diff
@@ -138,24 +141,28 @@ public int run(String[] args) {
         final ch.qos.logback.classic.Logger root = (ch.qos.logback.classic.Logger) rootLogger;
 
         if (cmd.hasOption('l')) {
-            LoggerContext loggerContext = (LoggerContext) LoggerFactory.getILoggerFactory();
-            loggerContext.reset();
-
-            PatternLayoutEncoder ple = new PatternLayoutEncoder();
-            ple.setPattern("%mdc{LOG-CONTEXT}%level %logger{36} %msg%n");
-            ple.setContext(loggerContext);
-            ple.start();
-
-            FileAppender<ILoggingEvent> fileAppender = new FileAppender<>();
-            fileAppender.setEncoder(ple);
-            fileAppender.setContext(loggerContext);
-            fileAppender.setName("fileLogging");
-            fileAppender.setAppend(false);
-            fileAppender.setFile(cmd.getOptionValue("l"));
-            fileAppender.start();
-
-            root.addAppender(fileAppender);
-            root.setLevel(Level.INFO);
+            if (runningInContainer()) {
+                logger.error("Disabling log file as running in container!");
```

Comment:
I would log it as warn rather than error.

Reply:
The reason I put it at error (which it is in https://github.com/release-engineering/pom-manipulation-ext/blob/master/cli/src/main/java/org/commonjava/maven/ext/cli/Cli.java#L179 ) is because it was a error by the user to attempt to enable logging inside the container. But I see your argument that you would prefer a warning logged instead - I guess it can be argued both ways :-)

Reply:
@pkocandr I've changed it to warn as requested.